src/components/hierarchy-bar/hierarchy-bar-collapsed/hierarchy-bar-collapsed.component.html

```diff
@@ -11,23 +11,22 @@
         (select)="hierarchyBar.selectNode($event)">
     </ux-hierarchy-bar-node>
 
-    <div class="hierarchy-bar-overflow" *ngIf="_links.length > 0">
+    <div class="hierarchy-bar-overflow" *ngIf="_parents.length > 0">
         <div class="hierarchy-bar-overflow-container">
             <ng-container [ngTemplateOutlet]="hierarchyBar.overflowTemplate || defaultOverflowTemplate"></ng-container>
         </div>
 
-        <button aria-label="Show children"
+        <button aria-label="Show siblings"
```

Comment:
This string is localizable - could you add an input property to configure it?

Reply:
Well spotted, added input to configure this